samples/outlook/10-roaming-settings/roaming-settings.yaml

```
@@ -58,36 +58,79 @@ template:
             <label class="ms-Label">Setting value:</label>
             <input id="settingValue" class="ms-TextField-field" type="text" value="world" placeholder="">
```

```suggestion
            <input id="settingValue" type="text" value="world" placeholder="">
```
Here and other instances.